sdk/src/main/aidl/com/robotemi/sdk/ISdkService.aidl

```diff
@@ -178,4 +178,8 @@ interface ISdkService {
     boolean getPrivacyModeState();
 
     void constraintBeWith();
+
+    void toggleHardButtons(in boolean enabled);
```

Comment:
change to disableHardButtons(in boolean disable)

The main purpose of this method is to disable the hard buttons not have the ability to toggle between the modes, also it makes the purpose of the method clearer, when the method name is toggle the users don't quite know what you are toggling between.